packages/template-block/src/TemplateBlock.spec.ct.tsx

```diff
@@ -79,6 +79,20 @@ describe('Template Block', () => {
         cy.get(CTA_BUTTON_SELECTOR).should('exist');
     });
 
+    it('should not render a block if use is not authenticated', () => {
+        const [TemplateBlockWithStubs] = withAppBridgeBlockStubs(TemplateBlock, {
+            blockTemplates: {
+                template: [getTemplateDummyWithPages()],
+            },
+            user: UserDummy.with(0),
+        });
+
+        mount(<TemplateBlockWithStubs />);
+        cy.get(BLOCK_CONTAINER_SELECTOR).should('exist');
+        cy.wait(100);
```

Comment:
This wait is there to finish the React rendering first. I noticed that `.should('exist')` AND `.should('not.exist')` both were true - because Cypress was quicker than the React rendering, so first the div was not there, then it was. 😅

Reply:
Maybe we should migrate these tests to `vitest` (What we gonna do for new blocks anyway...)

Reply:
Definitely! I wasn't aware of the state here.. As it's only one file anyway I assume it's not gonna be extremely difficult to do that. Let's do that as a future enhancement.